packages/lightning-lsp-common/src/context.ts

```diff
@@ -449,6 +457,29 @@ export class WorkspaceContext {
         }
     }
 
+    private async writeTsconfigJson(): Promise<void> {
+        switch (this.type) {
+            case WorkspaceType.SFDX:
+                // Write tsconfig.sfdx.json first
+                const baseTsConfigPath = path.join(this.workspaceRoots[0], '.sfdx', 'tsconfig.sfdx.json');
+                const baseTsConfig = await fs.readFile(utils.getSfdxResource('tsconfig-sfdx.base.json'), 'utf8');
+                this.updateConfigFile(baseTsConfigPath, baseTsConfig);
+                // Write to the tsconfig.json in each module subdirectory
+                const tsConfigTemplate = await fs.readFile(utils.getSfdxResource('tsconfig-sfdx.json'), 'utf8');
+                const forceignore = path.join(this.workspaceRoots[0], '.forceignore');
+                // TODO: We should only be looking through modules that have TS files
+                const modulesDirs = await this.getModulesDirs();
```

Comment:
In a follow-up PR I plan to only check for `lwc` directories that actually have TS files.

For now, I'm writing to all `lwc` directories.

Reply:
We should also only be generating either a `jsconfig.json` or a `tsconfig.json` but not both, this can be done once we've moved to GA.